### PR TITLE
HLTV: Fix crash and small refactoring.

### PR DESCRIPTION
**Pull request by s1lentq**
HLTV: Removed the obsolete filesystem_steam.

### CODE REVIEW

File: rehlds/HLTV/Core/src/World.cpp [hunk 1]
```diff
@@ -1060,65 +1060,65 @@ void World::ParseClientData(BitBuffer *stream, unsigned int deltaSeqNr, BitBuffe
...
-		else if (stream->ReadBit() == 0)
-		{
-			num = stream->ReadBits(6) + *numbase;
-		}
-		else
-		{
-			num = stream->ReadBits(11);
-		}
+		isdelta = stream->ReadBit() ? true : false;
+		remove = false;
```
Comment (In-line) [inline rehlds/HLTV/Core/src/World.cpp]: "remove" is incorrectly highlighted in Github, mb rename it?

Comment (WPMGPRoSToTeMa) [inline rehlds/HLTV/Core/src/World.cpp]: @In-line `isRemove`?

Comment (In-line) [inline rehlds/HLTV/Core/src/World.cpp]: @WPMGPRoSToTeMa, I think it's good. I also emailed Github support about this.

Comment (IgnacioFDM) [inline rehlds/HLTV/Core/src/World.cpp]: I don't think variable names should be decided based on github's syntax highlighting. No need to change it IMO

Comment (LevShisterov) [inline rehlds/HLTV/Core/src/World.cpp]: +1 to the 

> variable names shouldn't be decided based on github's syntax highlighting

But variable still can be renamed.

**Pull request merged**